playground/loggo/index.html

```diff
@@ -279,10 +279,11 @@ <h2 id="feed-heading">Feed <button id="refresh-sign" class="secondary span">Refr
             )
         );
         for (let instance of instances) {
+            if (!Array.isArray(instance?.feed)) continue
             for (let post of instance.feed) {
                 const [status, username, timestamp, gif] = post;
                 const isGifValid = /^(tode|berd|bot)$/.test(gif);
-                if (new Date(timestamp) > new Date()) {
+                if (new Date(new Date(post.date - 18000000)) > new Date()) {
```

Comment:
What's the reasoning for this magic number?

Reply:
5 hours in milliseconds (Login appears to save statuses with a date 5 hours in advance)

Reply:
ok cool! maybe worth a comment :)

Reply:
Updated 👍